worker/src/durable.rs

```diff
@@ -194,6 +194,33 @@ impl State {
     pub fn _inner(self) -> DurableObjectState {
         self.inner
     }
+
+    pub fn accept_web_socket(&self, ws: WebSocket) {
+        self.inner.accept_websocket(ws.as_ref().clone())
+    }
+
+    pub fn accept_websocket_with_tags(&self, ws: WebSocket, tags: &[&str]) {
+        let tags = tags.iter().map(|it| (*it).into()).collect();
+
+        self.inner
+            .accept_websocket_with_tags(ws.as_ref().clone(), tags);
+    }
```

Comment:
Could this be a reference (`&WebSocket`) instead? It's a paper-cut for the caller (who may need to `clone` the websocket), and then this function immediately derefs and clones again...

In fact, I wonder if this could be a reference on the inner layer as well.

Reply:
Yeah good catch, I'm really not sure about ownership when it comes to `wasm_bindgen`. For what it's worth `WebSocket` is just a single u32 so cloning it a lot is extremely cheap as far as I know.